types/ApiRequests.ts

```diff
@@ -74,6 +79,14 @@ export interface UpdateFeedbackStatusRequest {
   status: FeedbackStatus;
 }
 
+export interface InsertUserSocialMediaRequest {
+  socialMedia: SocialMedia;
+}
+
+export interface UpdateUserSocialMediaRequest {
+  socialMedia: Partial<SocialMedia>;
+}
```

Comment:
I think this should be a custom `SocialMediaPatches` type rather than `Partial<SocialMedia>`, since using `SocialMedia` as a partial means the `type` property is also allowed, which it shouldn't be, since users shouldn't be able to update the social media type of their URLs.